### PR TITLE
python: define test workflow

### PR DESCRIPTION
**Pull request by crockeo**
Description: Sets up the GH workflow to execute python tests in CI.
Risk Level: Low
Testing: This!
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

**Comment by goaway** [comment]
Can you fix the test on this PR too, please.

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: python
```
Comment (goaway) [inline .github/workflows/python.yml:1]:
```suggestion
name: python_tests
```

File: .github/workflows/python.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+on:
+  push:
+    branches:
+      - main
+    pull_request:
+
+jobs:
+  swifttests:
```
Comment (goaway) [inline .github/workflows/python.yml:10]:
```suggestion
pythontests:
```

**Review by goaway — APPROVED** [review]

**Comment by crockeo** [comment]
closing this in favor of the version that exists on envoy-mobile, so we can test workflow

**Pull request closed without merging**